fkbeta/fkws/tests.py

```diff
@@ -331,6 +331,13 @@ def _get_upload_token_helper(self, thing_tests):
             self.assertEqual(status, r.status_code)
```

Comment:
A lot of these tests, that list "reverse names", should also have this new endpoint listed, i.e. `test_anonymous_cannot_add`, `test_nuug_user_can_edit_own_things`, `test_nuug_user_cannot_edit_nonowned_things`, `test_anonymous_cannot_edit` etc.

Reply:
Most of these functions work on -details views, not lists.  I did not plan to add the ability to add or update organizations using the API, are all these still needed also for lists?